app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+    for i in people:
+        result.append(Person(i["name"], i["age"]))
+    for i in people:
+        person = Person.people.get(i["name"])
+        if "wife" in i and i["wife"]:
+            wife_person = Person.people.get(i["wife"])
+            person.wife = wife_person
+            wife_person.husband = person
```

Comment:
These lines assume that `wife_person` and `person` have `wife` and `husband` attributes, respectively. However, these attributes are not defined in the `Person` class. Consider initializing these attributes in the `__init__` method or handling the case where they do not exist.